3512-0/triangle.rb

```diff
@@ -0,0 +1,29 @@
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+def triangle(var_a, var_b, var_c)
+  side = [var_a, var_b, var_c].sort
+
+  raise TriangleError, 'No negative' if side.any? { |s| s <= 0 }
```

Comment:
You already have sorted array, if min element is positive maybe we don't need to check something else?